.github/workflows/build_packages_cron.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
         profile:
           - ['emqx', 'master']
           - ['emqx', 'release-58']
+          - ['emqx-enterprise', 'release-58']
```

Comment:
Q: does this change take effect if not present on `master`?

Reply:
Right, it does not.